### PR TITLE
feat: to keep controller function attributes

### PR DESCRIPTION
**Pull request by gxcsoccer**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

**Comment by dead-horse** [comment]
3.1.0

File: lib/loader/mixin/controller.js [hunk 1]
```diff
@@ -86,16 +86,24 @@ function wrapObject(obj, path) {
         throw new Error(`controller \`${key}\` should not use next as argument from file ${path}`);
       }
       ret[key] = functionToMiddleware(obj[key]);
+    } else if (is.object(obj[key])) {
+      ret[key] = wrapObject(obj[key], path);
```
Comment (popomore) [inline lib/loader/mixin/controller.js]: 这个地方嵌套多层并非是最佳实践，而且挺容易冲突的

Comment (dead-horse) [inline lib/loader/mixin/controller.js]: 刚才没看到这里，这个有使用场景的么？

Comment (dead-horse) [inline lib/loader/mixin/controller.js]: ping @gxcsoccer

**Pull request merged**